apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Tests/Log4NetTest.cs

```diff
@@ -38,7 +39,7 @@ public class Log4NetTest
         [MethodImpl(MethodImplOptions.NoInlining)]
         private void LogInfo(IEnumerable<string> messages)
         {
-            ILog log = LogManager.GetLogger("testlogger");
+            ILog log = LogManager.GetLogger(Assembly.GetEntryAssembly(), "testlogger");
```

Comment:
`Assembly.GetEntryAssembly()` returns null on netcore, causing tests to fail. Please use `typeof(Log4NetTest).GetTypeInfo().Assembly`.